src/exercism/Leap.java

```diff
@@ -0,0 +1,10 @@
+class Leap {
+    boolean isLeapYear(int year) {
+       return year % 4 == 0 && year % 100 != 0 || year % 400 == 0;
```

Comment:
I think it  is more readable if you write  `(year % 4 == 0 && year % 100 != 0) || year % 400 == 0`

So that it is more clear what operation you run first